src/proposals/20231115/SparkEthereum_20231115.sol

```diff
@@ -0,0 +1,159 @@
+    address public constant GNOSIS_FORWARDER = 0x44f993EAe9a420Df9ffa5263c55f6C8eF46c0340;
+    address public constant GNOSIS_PAYLOAD   = 0xF62849F9A0B5Bf2913b396098F7c7019b51A820a; // Temporary test address, to be replaced after deployment
+
+    function capsUpdates()
+        public pure override returns (IEngine.CapsUpdate[] memory)
+    {
+        IEngine.CapsUpdate[] memory capsUpdate = new IEngine.CapsUpdate[](3);
+        capsUpdate[0] = IEngine.CapsUpdate({
+            asset:     RETH,
+            supplyCap: 80_000,
+            borrowCap: EngineFlags.KEEP_CURRENT
+        });
+
+        capsUpdate[1] = IEngine.CapsUpdate({
+            asset:     WSTETH,
+            supplyCap: 800_000,
+            borrowCap: EngineFlags.KEEP_CURRENT
+        });
+
+        capsUpdate[2] = IEngine.CapsUpdate({
+            asset:     WBTC,
+            supplyCap: 3_000,
+            borrowCap: 2_000
+        });
+
+        return capsUpdate;
+    }
+
+    function collateralsUpdates()
+        public pure override returns (IEngine.CollateralUpdate[] memory)
+    {
+        IEngine.CollateralUpdate[] memory collateralUpdates = new IEngine.CollateralUpdate[](2);
+
+        collateralUpdates[0] = IEngine.CollateralUpdate({
+            asset:          DAI,
+            ltv:            0,
+            liqThreshold:   EngineFlags.KEEP_CURRENT,
+            liqBonus:       EngineFlags.KEEP_CURRENT,
+            debtCeiling:    EngineFlags.KEEP_CURRENT,
+            liqProtocolFee: EngineFlags.KEEP_CURRENT,
+            eModeCategory:  EngineFlags.KEEP_CURRENT
+        });
+
+        collateralUpdates[1] = IEngine.CollateralUpdate({
+            asset:          WBTC,
+            ltv:            70_00,
+            liqThreshold:   75_00,
+            liqBonus:       7_00,
+            debtCeiling:    0,
+            liqProtocolFee: 10_00,
+            eModeCategory:  0
+        });
+
+        return collateralUpdates;
+    }
+
+    function borrowsUpdates()
+        public pure override returns (IEngine.BorrowUpdate[] memory)
+    {
+        IEngine.BorrowUpdate[] memory borrowsUpdate = new IEngine.BorrowUpdate[](1);
+
+        borrowsUpdate[0] = IEngine.BorrowUpdate({
+            asset:                 WBTC,
+            reserveFactor:         20_00,
+            enabledToBorrow:       EngineFlags.ENABLED,
+            flashloanable:         EngineFlags.ENABLED,
+            stableRateModeEnabled: EngineFlags.DISABLED,
+            borrowableInIsolation: EngineFlags.DISABLED,
+            withSiloedBorrowing:   EngineFlags.DISABLED
+        });
+
+        return borrowsUpdate;
+    }
+
+    function rateStrategiesUpdates()
+        public pure override returns (IEngine.RateStrategyUpdate[] memory)
+    {
+        IEngine.RateStrategyUpdate[] memory ratesUpdate = new IEngine.RateStrategyUpdate[](2);
+
+        Rates.RateStrategyParams memory wethRateStrategyParams = Rates.RateStrategyParams({
+            optimalUsageRatio:             _bpsToRay(90_00),
```

Comment:
optimalUsageRatio isn't supposed to change also, so can just update variableRateSlope and baseVariableBorrowRate only